volumina/imageScene2D.py

```diff
@@ -425,7 +425,7 @@ def drawBackground(self, painter, sceneRectF):
             for through in self._bowWave(self._n_preemptive):
                 self._tileProvider.prefetch(sceneRectF, through)
 
-    def joinRenderingAllTiles(self, viewport_only=True):
+    def joinRenderingAllTiles(self, viewport_only=True, rect=None):
         """
```

Comment:
Now you're using this function for non-test purposes.  That's fine, but please update the docstring here.